src/JoomlaBrowser.php

```diff
@@ -684,13 +684,16 @@ public function createMenuItem($menuTitle, $menuCategory, $menuItem, $menu = "Ma
         $I = $this;
         $I->amOnPage('administrator/index.php?option=com_menus&view=menus');
         $I->waitForText('Menus', '30', ['css' => 'H1']);
+		$this->checkForPhpNoticesOrWarnings();
 
         // Choose the right menu
         $I->click(['xpath' =>  "//*[@id=\"menuList\"]/tbody/tr/td[2]/a[contains(text(), '" . $menu . "')]"]);;
         $I->waitForText('Menus: Items', '30', ['css' => 'H1']);
+		$this->checkForPhpNoticesOrWarnings();
 
-        $I->click(['xpath' => "//button[@onclick=\"Joomla.submitbutton('item.add')\"]"]);
+        $I->click("New");
```

Comment:
Checked, as you said, it works:

![screen shot 2015-10-01 at 11 58 50](https://cloud.githubusercontent.com/assets/1375475/10217979/cc60ce72-6833-11e5-9877-9591a3dfd607.png)

Without issues, it uses the Codeception heuristics witch could be slow but seems pretty fast. What it fails is the `Save & Close` for some unknown reason. 


![screen shot 2015-10-01 at 11 58 14](https://cloud.githubusercontent.com/assets/1375475/10217993/eebbfa28-6833-11e5-83d4-78c6b8032fb6.png)

But since we are not using it here I'm merging: